### DISCUSSION TITLE
Use Tfsec custom Checks in Trivy

### DISCUSSION THREAD

**Discussion by volturyon** [aquasecurity/trivy#5421] [Q&A] [answered]
### Question

Hello! I'm trying to use my custom checks created in yaml for tfsec in trivy and I still haven't found a way. In a previous issue they commented that it is possible but I can't find the key. 
Any ideas? 

Thanks!

https://github.com/aquasecurity/trivy/issues/1802

### Target

None

### Scanner

Misconfiguration

### Output Format

None

### Mode

None

### Operating System

MacOs

### Version

```bash
trivy --version
Version: 0.45.1
Vulnerability DB:
  Version: 2
  UpdatedAt: 2023-09-20 06:22:56.535814102 +0000 UTC
  NextUpdate: 2023-09-20 12:22:56.535813202 +0000 UTC
  DownloadedAt: 2023-09-20 08:20:30.404711 +0000 UTC
Policy Bundle:
  Digest: sha256:1df8ade71efc830877ca3b1130f83e0c6368e3a45b0d4c0f0418955501644054
  DownloadedAt: 2023-10-21 18:56:29.977868 +0000 UTC
```

**Comment by simar7** [comment]
Please find details here https://aquasecurity.github.io/trivy/v0.46/docs/scanner/misconfiguration/custom/

**Reply by volturyon** [reply]
Thank you very much for your reply! But on that page I can only find how to write them in rego, in my case all the custom checks of tfsec I had written in yaml. Is there no way to use them? It would make the migration much easier

**Reply by simar7** [reply, marked as answer]
Unfortunately we only support Rego at the moment for writing custom checks.

**Reply by volturyon** [reply]
> Unfortunately we only support Rego at the moment for writing custom checks. 

Thank you so much! I hope that in the near future it can be implemented, since yaml is extremely comfortable for me
